nilearn/reporting/_plot_matrices.py

```diff
@@ -62,6 +64,94 @@ def plot_design_matrix(design_matrix, rescale=True, ax=None, output_file=None):
     return ax
 
 
+def plot_event(model_event, cmap=None, output_file=None, **fig_kwargs):
+    """Creates plot for event visualization.
+
+    Parameters
+    ----------
+    model_event : pandas DataFrame or list of pandas DataFrame
+        the `pandas.DataFrame` must have three columns
+        ``event_type`` with event name, ``onset`` and ``duration``.
+        The `pandas.DataFrame` can also be obtained from 
+        :func:`~nilearn.stats.first_level_model.first_level_models_from_bids`.
+
+    cmap : str or matplotlib.cmap, optional
+        the colormap used to label different events
+
+    output_file : string or None, optional,
+        The name of an image file to export the plot to. Valid extensions
+        are .png, .pdf, .svg. If output_file is not None, the plot
+        is saved to a file, and the display is closed.
+
+    **fig_kwargs : extra keyword arguments, optional
+        Extra arguments passed to matplotlib.pyplot.subplots
+
+    Returns
+    -------
+    Plot Figure object
+
+    """
+
+    if isinstance(model_event, pd.DataFrame):
+        model_event = [model_event]
+   
+    n_runs = len(model_event)
+    figure, ax = plt.subplots(1, 1, **fig_kwargs)
+
+    # input validation
+    if cmap is None:
+        cmap = plt.cm.tab20
+    elif isinstance(cmap, str):
+        cmap = plt.get_cmap(cmap)
+    else:
+        cmap = cmap
+
+    event_labels = pd.concat(e['trial_type'] for e in model_event)
```

Comment:
```suggestion
    event_labels = pd.concat(event['trial_type'] for event in model_event)
```Bzw, I'm surprise that this works. Thought that we would need `[event['trial_type'] for event in model_event]`

Reply:
I read it few days ago from a colleague and it worked :)!!
I think that In some cases it is more readable but it is not straightforward that you are passing a list!